guvnor-m2repo-editor/guvnor-m2repo-editor-client/src/main/java/org/guvnor/m2repo/client/upload/UploadFormViewImpl.java

```diff
@@ -195,8 +146,80 @@ private String getWebContext() {
         return context;
     }
 
-    protected void showUploadingBusy() {
-        // LoadingPopup.showMessage( "Uploading...");
+    public void showUploadingBusy() {
+        // TODO: "Uploading..." needs localization
+        BusyPopup.showMessage("Uploading...");
```

Comment:
@manstis Yes, I am tested it at runtime as well as run new tests  and it's works as expected.
Ok, thank you I will make this changes.